backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -179,12 +179,23 @@ export class ContributionResolver {
   async listAllContributions(
     @Args()
     { currentPage = 1, pageSize = 5, order = Order.DESC }: Paginated,
+    @Arg('statusFilter', () => [ContributionStatus], { nullable: true })
```

Comment:
Is this working?
```suggestion
    @Arg('statusFilter', () => [ContributionStatus!], { nullable: true })
```

Reply:
Is not working for me, it is writting the following error:
```
Forbidden non-null assertion.eslint[@typescript-eslint/no-non-null-assertion](https://github.com/typescript-eslint/typescript-eslint/blob/v4.33.0/packages/eslint-plugin/docs/rules/no-non-null-assertion.md)
```
I could add a line with
```
    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```
But i am not sure if we want it.

Reply:
OK. Thanks for trying. Leave it as it is